server/modules/google-analytics.js

```diff
@@ -0,0 +1,11 @@
+let GoogleAnalytics = (() => {
```

Comment:
Also I think this module's instantiation is too fancy.

```javascript
function getGoogleAnalyticsScript(tagId) {
   return ...
}

modules.exports = {
   getGoogleAnalyticsScript,
}
```

I believe the example above do the same job.

Reply:
Thanks for the example, I was looking at other modules and trying to fit in with other existing code.